scripts/weather.coffee

```diff
@@ -0,0 +1,36 @@
+# Description:
+#   Look up weather information
+#
+# Commands:
+#
+#   hubot weather <location> - look up weather at location (in google-understandable format)
+#
+# Configuration:
+#
+# HUBOT_WEATHER_APIKEY = forecast.io api key
+module.exports = (robot) ->
+
+  robot.respond /weather *(.+)/i, (msg) ->
+    location = msg.match[1]
+    q = address: '#{location}'
```

Comment:
`q` is unused here, since you interpolate directly into the URL below. Mind deleting it?